OWNERS_ALIASES

```diff
@@ -1,20 +1,44 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 aliases:
-  approvers:
-      - davidvossel
+  sme-live-migration:  # Experts on live migration
       - vladikr
-      - rmohr
-      - stu-gott
-      - fabiand
-      - AlonaKaplan
-      - dhiller
+      - acardace
       - jean-edouard
-      - mhenriks
-      - enp0s3
       - xpivarc
       - vasiliy-ul
+      - iholder
+
+  sme-api-n-controllers:  # Experts on API and Controller mechanics
+      - rmohr
+      - davidvossel
       - acardace
+      - jean-edouard
+      - xpivarc
+      - vasiliy-ul
+
+  sme-scheduling-n-node-plumbing:  # Experts on Scheduling and Node plumbing
+      - vladikr
+      - jean-edouard
+      - xpivarc
+      - vasiliy-ul
+      - AlonaKaplan
+
+  sme-scale-n-observability:  # Experts on Scale and Observability
+      - rthallisey
+      - sradco
+      - davidvossel
+
+  sme-ci-n-testing:  # Experts on CI and Testing
+      - enp0s3
+      - jean-edouard
+      - dhiller
+      - stu-gott
+
+  sme-buildsystem:  # Experts on the build-system
+      - enp0s3
+      - dhiller
+
```

Comment:
Maybe add David here, since he does the release?
```suggestion
      - davidvossel

```

Reply:
Yes, sounds fair ... although I wonder if it should then be build-system-n-releases